common/v2/components/Banner.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Banner } from './Banner';
+import { BannerType } from '../types';
+
+export default { title: 'Banner' };
+
+export const announcement = () => (
+  <Banner value="I'm an announcement banner!" type={BannerType.ANNOUNCEMENT} />
+);
+
+// Uncomment this for Figma support:
```

Comment:
Should this be included?

Reply:
@vilsbole asked for Figma support, so this is just an example of how to use it.

Reply:
It would be nice to see the react component side-by-side figma for comparing.